### PR TITLE
Formalize some digitizing tools instructions as steps

### PR DESCRIPTION
**Pull request by DelazJ**
Also add precision and fixes typos

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 1]
```diff
@@ -1096,33 +1110,40 @@ attributes are made identical.
...
+   #. In the :menuselection:`Layer Properties --> Symbology` dialog, browse to
+      the symbol editor dialog.
+   #. Click the |dataDefined| :guilabel:`Data-defined override` widget near the
+      :guilabel:`Rotation` option of the top :guilabel:`Marker` level (preferably)
+      of the symbol layers. 
+   #. Choose a field in the :guilabel:`Field Type` combobox. Values of this
+      field are hence used to rotate each feature's symbol accordingly.
 
-.. _figure_rotate_point:
+   .. note:: **Ensure to assign the same field to all the symbol layers**
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: Make sure that the same field is assigned to all the symbol layers

Comment (DelazJ) [reply source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: @havatv tip: try github suggestion feature. 😄 Speeds up correction.

**Comment by DelazJ** [comment]
Thanks for the review @havatv Testing have been done weeks ago (and this week-end for point symbols) so should be good. Though the main changes are formatting matters.

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst (hunk 1 above)
Comment (havatv) [reply source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: Sounds like a good idea :-). I will try to remember.

**Pull request merged**